src/core/src/window.rs

```diff
@@ -141,7 +142,7 @@ pub trait Surface<B: Backend> {
     fn build_swapchain<C>(&mut self,
         config: SwapchainConfig,
         present_queue: &CommandQueue<B, C>,
-    ) -> (B::Swapchain, Vec<Backbuffer<B>>);
+    ) -> (B::Swapchain, Backbuffer<B>);
```

Comment:
FWIW my goal for the future is to only return the swapchain here and add a function for retrieving the images (with the hack for GL). The GL backend would additionally provide a function to retrieve the default framebuffer.